recipes/psiflow/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "psiflow" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molmod/psiflow/archive/v{{ version }}.tar.gz
+  sha256: c6103905ecfe9ded67a1c52142424f184e50c24598998be931f25d5a13b6ae43
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - pmg = pymatgen.cli.pmg:main
+    - psiflow-train-nequip = psiflow.models.train_nequip:main
+    - psiflow-deploy-nequip = psiflow.models.deploy_nequip:main
+    - psiflow-train-mace = psiflow.models.train_mace:main
+    - psiflow-md-yaff = psiflow.walkers.molecular_dynamics_yaff:main
+    - psiflow-md-openmm = psiflow.walkers.molecular_dynamics_openmm:main
+    - psiflow-test = psiflow.test:main
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - ase >=3.22.1
+    - pyyaml >=6.0
+    - numpy >=1.22.3
+    - wandb >=0.14.0
+    - pymatgen >=2023.3.23
+    - parsl ==2023.08.28
+
+test:
+  imports:
+    - psiflow
+  requires:
+    - pip
+  commands:
```

Comment:
Is it possible to add `commands` tests for all of the entry points?

Reply:
I tested all six commands but was only able to get three of them to work.